lib/cms/prayers.ex

```diff
@@ -73,9 +86,33 @@ defmodule CMS.Prayers do
     %PrayerRequest{}
     |> PrayerRequest.changeset(attrs, scope)
     |> ensure_user_in_organization(scope)
+    |> validate_group_membership(scope)
     |> Repo.insert()
   end
 
+  defp validate_group_membership(changeset, %Scope{} = scope) do
```

Comment:
If we preload the groups in the scope user, we can move this validation to the prayer request changeset, right?